### ISSUE TITLE
Separated employee? causes error

### ISSUE THREAD

**Issue by terrordactylyou** [uci-soe/uci-directory-search#3]
```
dhcp191168:~ saclark$ d -i codgers
/usr/local/lib/node_modules/uci-directory-search/lib/search.js:39
            employee:      i.UCIaffiliation.indexOf('employee') !== -1,
                                           ^

TypeError: Cannot read property 'indexOf' of undefined
    at /usr/local/lib/node_modules/uci-directory-search/lib/search.js:39:44
    at Array.map (native)
    at /usr/local/lib/node_modules/uci-directory-search/lib/search.js:32:21
    at EventEmitter.<anonymous> (/usr/local/lib/node_modules/uci-directory-search/node_modules/uci-ldap-search/lib/index.js:62:7)
    at emitOne (events.js:77:13)
    at EventEmitter.emit (events.js:169:7)
    at _done (/usr/local/lib/node_modules/uci-directory-search/node_modules/uci-ldap-search/node_modules/ldapjs/lib/client/client.js:785:22)
    at messageCallback (/usr/local/lib/node_modules/uci-directory-search/node_modules/uci-ldap-search/node_modules/ldapjs/lib/client/client.js:869:16)
    at Parser.onMessage (/usr/local/lib/node_modules/uci-directory-search/node_modules/uci-ldap-search/node_modules/ldapjs/lib/client/client.js:199:12)
    at emitOne (events.js:77:13)
```

**Comment by rhettl** [comment]
So this person does still exist, If you look at her via `directory -i codgers` she still has many data points, but the lines determining affiliations (student, employee, etc) are not available any more. 

Instead of throwing a `SeparatedEmployeeError`, or something like that, I'm opting to change the above mentioned line from `employee: i.UCIaffiliation.indexOf('employee') !== -1,` to `employee: i.UCIaffiliation && i.UCIaffiliation.indexOf('employee') !== -1,` This will just make it say that she is not an employee as she has no affiliation.

**Comment by rhettl** [comment]
Fixed in PR #6
